website/docs/components/alert/partials/content/content.md

```diff
@@ -0,0 +1,151 @@
+## General recommendations
+
+- Keep the title short, as this will be the most prominent element when users scan the alert.
+- Avoid ending the title with a period.
+- Alert descriptions should be short but clear enough to explain what’s happening. We recommend keeping messages under 90 characters.
+- For warning and critical alerts, guide the users on how to prevent or fix the issue.
+
+## Actions
+
+Use small buttons to avoid competing with other actions on the page. Use more than two actions sparingly.
+
+We recommend using the `secondary` button variant for primary actions and the `tertiary` button variant for secondary actions.
+
+<Hds::Alert @type="inline" as |A|>
+  <A.Title>Recommended button usage</A.Title>
+  <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+  <A.Button @text="Your action" @color="secondary" @onClick={{this.noop}} />
+  <A.Button @color="tertiary" @icon="arrow-right" @iconPosition="trailing" @text="Tertiary" />
+</Hds::Alert>
+
+For content guidelines on actions, refer to [Button](/components/button) and 
+[Link](/components/link/standalone) documentation.
+
+### Usage of critical buttons
+
+Avoid using critical buttons in alerts. We handle the prominence and importance via the styling of the alert container itself. If you need to confirm that the user intended to interact with the action, consider displaying a confirmation modal.
+
+!!! Dont
+<Hds::Alert @type="inline" @color="critical" as |A|>
+  <A.Title>Critical button usage</A.Title>
+  <A.Description>Lorem ipsum dolar sit amet.</A.Description>
+  <A.Button @text="Your action" @color="critical" @onClick={{this.noop}} />
+</Hds::Alert>
+!!!
+
+### Links
```

Comment:
While reading this part, I was a bit confused on what links you (@jorytindall?) were referring to. IN some cases we're referring to the "actions/buttons" (that technically can be links as in `HTML anchors`) and in other cases to the inline links within the description text.
This I suspect may lead to confusion. For example, a user may think that it's OK to use a "standalone" link in the description (which is not).
Not sure how to improve this, but I think is worth another cycle of refinement.

Reply:
Yeah, that's a good identification of the challenge. I'll rev on this again

Reply:
I added a bit of clarity on a different comment, but it's applicable. Let me try to revise everything holistically.

Reply:
Perfect! 🙌